README.md

```diff
@@ -359,30 +361,33 @@ will kick off an asynchronous load of the specified media
 playlist. Once it has been retreived, it will become the active media
 playlist.
 
-#### hls.segmentXhrTime
+#### hls.systemBandwidth
 Type: `number`
 
-The number of milliseconds it took to download the last media segment.
-This value is updated after each segment download completes.
+`systemBandwidth` is a combination of two serial processes bit-rates. The first
```

Comment:
Gonna be super annoying: I think the grammar here is processes', given that it's plural/possessive.

Reply:
I encourage you to tear apart my grammar

Reply:
Also we can just do bitrates without the dash, since it is used elsewhere